runtime/bcutil/ConstantPoolMap.hpp

```diff
@@ -210,6 +210,13 @@ class ConstantPoolMap
 
 	U_16 getROMClassCPIndexForReference(U_16 cfrCPIndex)  const { return getROMClassCPIndex(cfrCPIndex); }
 	U_16 getROMClassCPIndexForAnnotation(U_16 cfrCPIndex) const { return getROMClassCPIndex(cfrCPIndex); }
+	U_16 getROMClassCPIndexForAnnotationWithCheck(U_16 cfrCPIndex) const { 
+		if ((getCPTag(cfrCPIndex) == CFR_CONSTANT_Utf8) && isMarked(cfrCPIndex, ANNOTATION)) {
+			 return getROMClassCPIndex(cfrCPIndex); 
+		} else {
+			return 0;
+		}
+	}
```

Comment:
Multiple returns are discouraged, especially with code this simple.  How about initializing a return value to 0, and setting it on success?

Reply:
Done.  Thanks.